packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -563,6 +580,8 @@ class _GoogleMapOptions {
 
   final bool tiltGesturesEnabled;
 
+  final bool tiltControlsEnabled;
```

Comment:
On web, you can only rotate after you toggle on 45 degree imagery (this seems to behave differently than maps.google.com, but that's probably not something we should be worrying about) which seems like a separate feature to me. At least on the projects I'm working on, we want rotation to work on android/ios, but we don't want users to be able to enable 45 degree imagery on web since that messes up our markers, polygons, etc.

Reply:
Going to have to see how the new web tilt/rotation features affect this

Reply:
> "we don't want users to be able to enable 45 degree imagery on web" 

@Rexios80: I agree. Since AFAIK the 45deg imagery is only available on the web, we could completely disable it by always passing:

```
    tilt: 0,
    rotateControl: false,
```

to the map options when creating a web map. ([Experiment here](https://jsfiddle.net/gh/get/library/pure/googlemaps/js-samples/tree/master/dist/samples/aerial-simple/jsfiddle)).

The tilt can still be controlled by gestures or programmatically, the above two parameters seem to force to never use a tilt != 0 imagery when in satellite mode (you can get it if you go to huge zoom levels without tilt:0) and to hide the "tilt 45 degrees" button, which enables the undesirable mode.

I think that is a sane default for the plugin, that makes the web implementation more consistent with the mobile one.

(We could also add a `webEnable45DegImagery` setting that does **not** apply the default, for the people who want/need it?)

Reply:
(This is being actively discussed in another thread, resolving this one!)